cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -347,7 +358,7 @@ spec:
             {{- end }}
             {{- end }}
             {{- end }}
-            {{- if .Values.oidc }}
+                        {{- if .Values.oidc }}
```

Comment:
Weird formatting thing happening here?

Reply:
nice find. not sure what happened there.